hydrolib/tools/extforce_convert/converters.py

```diff
@@ -871,6 +871,8 @@ def convert(
             "xcoordinates": polyline.x,
             "ycoordinates": polyline.y,
         }
+        if forcing.area is not None:
+            data["area"] = forcing.area
```

Comment:
is there any other attributes that could be passed from the Forcing instance or just the `area` was missing ?